packages/math-input/src/components/keypad/operators-page/operators-page.tsx

```diff
@@ -1,16 +1,24 @@
 import * as React from "react";
 
-import Keys from "../../data/key-configs";
-
+import Keys from "../../../data/key-configs";
 import {
     KeypadPageContainer,
     SecondaryKeypadButton,
     KeypadActionButton,
     PlaceHolderButton,
-} from "./keypad-page-items";
+} from "../keypad-page-items";
+
+import {PreAlgebra} from "./pre-algebra-buttons";
 
 type Props = {
     onClickKey: (keyConfig: string) => void;
+} & OperatorsButtonSets;
+
+export type OperatorsButtonSets = {
+    preAlgebra?: boolean;
+    logarithms?: boolean;
+    relations?: boolean;
+    advancedRelations?: boolean;
```

Comment:
Could we move this into a `types.ts` file?  As soon as other files start importing types from another component file, it starts to feel "off" to me.  It also avoids circular imports in the future.